x/rollapp/keeper/keeper.go

```diff
@@ -21,22 +21,36 @@ type Keeper struct {
 
 	ibcClientKeeper types.IBCClientKeeper
 	channelKeeper   types.ChannelKeeper
+
+	finalizePending func(ctx sdk.Context, stateInfoIndex types.StateInfoIndex) error
```

Comment:
why we need this as an interface?

Reply:
To be able to mock it easily, for returning errors on certain rollapp state indexes in tests